File: Invalid Input - Error Handling #1.py
import re
def get_count(words=''):
    dic = {'vowels':0,'consonants':0}
    if type(words)==str:
        words = re.sub(r'[^a-zA-Z]',"",words)
        print(words)
        for word in words.lower():
            if word in ['a','i','u','e','o']:
                dic['vowels']+=1
            else:
                dic['consonants']+=1
        return dic
    else:
        return dic

File: Invalid Input - Error Handling #1/Invalid Input - Error Handling #1.js
const getCount = (words) => {
  let dic = {"vowels":0,"consonants":0}
  if(typeof(words)!=='string'){
    return dic
  }
  words = words.replace(/[^a-z]/ig,"").toLowerCase()
  words.split("").map(v => {
     if(/[aiueo]/.test(v)){
       dic['vowels']+=1
     }else{
       dic['consonants']+=1
     }
  })
  return dic
}
